pkg/logging/syslog/syslog_integration_test.go

```diff
@@ -0,0 +1,449 @@
+package syslog_test
```

Comment:
This is a direct `mv syslog_test.go syslog_integration_test.go`.

Note that fields like `Hostname` and `IPV4` exist, but can't be set.

This is addressed in a separate branch.